performance/PerformanceBaselines.json

```diff
@@ -2,77 +2,103 @@
     "PerformanceRuntime": {
         "InputBinding": {
             "TestName": "GetProductsTest",
-            "RunTimeAverageSec": 2.081
+            "RunTimeAverageSec": 2.081,
+            "RunTimeAverageMinSec": 2.208,
```

Comment:
These seem to be reversed. And also is this supposed to be 10%? The numbers seem off if so - for example

2.081 + (0.1 * 2.081) = 2.2891, not 2.208